README.md

```diff
@@ -86,6 +86,18 @@ binary_sensor:
 
 By adding this sensor, you will get a boolean sensor in Home Assistant indicating the current state of the garage door motor light.
 
+### Configure battery sensor
+
+```
+# Add if you want a binary sensor to show if the dc blue unit is running off backup battery
+binary_sensor:
+  - platform: dc_blue
+    battery:
+      name: "Garage Motor on Battery Backup"  
+```
+
+By adding this sensor, you will get a boolean sensor in Home Assistant indicating the current state of the power supply.
```

Comment:
"Garage Motor on Battery Backup" is maybe a little ambiguous. Does it mean the motor has a battery installed, or does it mean AC power is off?

If it's the latter, perhaps we should invert the logic and rename it to "Garage Motor AC supply" -> ON/OFF.

Reply:
Thanks for the feedback. That probably makes a lot more sense. Will update this evening

Reply:
@jpmeijers I have made the suggested changes